inscriptions/ISic003710.xml

```diff
@@ -161,18 +165,19 @@
             </langUsage>
             <textClass>
                 <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
-                    <term ana="#function.funerary" ref="http://www.eagle-network.eu/voc/typeins/lod/92.html">funerary</term>
+                    <term ana="#function.funerary function.unknown" ref="http://www.eagle-network.eu/voc/typeins/lod/92.html">funerary, other</term>
```

Comment:
second function needs preceding #; but actually this is a certainty issue and alternative, not a hierarhical or cumulatirve?alternatively just make it funerary and cert="low"?

Reply:
I use cert="low" and add as plain text 'funerary (?), writing exercise (?)'. Let me know if this is ok. Then we can add in the commentary more information...